features/gba-dev/kinderen.feature

```diff
@@ -0,0 +1,146 @@
+    - Wanneer een gegeven een standaardwaarde heeft, zoals "." (punt) bij geslachtsnaam of "00000000" bij geboortedatum, geldt dat hier als het bestaan van een waarde en wordt het kind wel geleverd
+    - Wanneer door de gebruikte fields parameter in het request het kind in de response geen enkel gegeven heeft met een waarde, dan wordt het kind geleverd zonder gegevens (dus als leeg object)
+
+    Scenario: kind volledig onbekend, veld met onbekend waarde wordt gevraagd met fields
+      Gegeven de persoon met burgerservicenummer '000000036' heeft een 'kind' met de volgende gegevens
+      | geslachtsnaam (02.40) | geboortedatum (03.10) | 
+      | .                     | 00000000              |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000036                       |
+      | fields              | kinderen.naam                   |
+      Dan heeft de response een persoon met een 'kind' met de volgende 'naam' gegevens
+      | naam          | waarde |
+      | geslachtsnaam | .      |
+	  
+    Scenario: kind volledig onbekend, veld met onbekend waarde wordt niet gevraagd met fields
+      Gegeven de persoon met burgerservicenummer '000000048' heeft een 'kind' met de volgende gegevens
+      | geslachtsnaam (02.40) | geboortedatum (03.10) | 
+      | .                     | 00000000              |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000048                       |
+      | fields              | kinderen.burgerservicenummer    |
+      Dan heeft de response een persoon met een 'kind' zonder gegevens
+
+    Scenario: ontkenning ouderschap
+      Gegeven de persoon met burgerservicenummer '000000061' heeft een 'kind' met de volgende gegevens
+      | voornamen (02.10) | voorvoegsel (02.30) | geslachtsnaam (02.40) | geboortedatum (03.10) | gemeente document (82.10) | datum document (82.20) | beschrijving document (82.30) | ingangsdatum geldigheid (85.10) | datum van opneming (86.10) |
+      | Daan              | de                  | Vries                 | 20031107              | 0518                      | 20031109               | PL gerelateerde               | 20031107                        | 20031109                   |
+      En het 'kind' is gewijzigd naar de volgende gegevens
+      | gemeente document (82.10) | datum document (82.20) | beschrijving document (82.30) | ingangsdatum geldigheid (85.10) | datum van opneming (86.10) |
+      | 1926                      | 20040105               | D27894-2004-A782              | 20031107                        | 20040112                   |
```

Comment:
Wat maakt nu dat dit een ontkenning ouderschap is ? 
Dit zou toch ook een document kunnen zijn dat met Adoptie te maken heeft ?

Reply:
> Wat maakt nu dat dit een ontkenning ouderschap is ? 

Goeie vraag aangezien het kind wel blijft bestaan als kind van de persoon met bsn 0000000061, van enkele velden (naamgegevens, geboortedatum) wordt alleen de waarde verwijderd.

> Dit zou toch ook een document kunnen zijn dat met Adoptie te maken heeft ?

Dan zou je de waarde van de velden toch niet verwijderen?

Is het trouwens van belang voor dit scenario dat de gemeentecode bij 'gemeente document (82.10)' wordt gewijzigd van '0518' naar '1926'?

Wellicht nog in de documentatie van de e2e automation aangeven dat als een veld in een 'gewijzigd' step niet wordt opgegeven terwijl het in de Gegeven step wel is opgevoerd de waarde is verwijderd.

Reply:
Ik heb de tabellen verticaal gemaakt zodat ze wat beter te lezen zijn.

Reply:
> 
> Wellicht nog in de documentatie van de e2e automation aangeven dat als een veld in een 'gewijzigd' step niet wordt opgegeven terwijl het in de Gegeven step wel is opgevoerd de waarde is verwijderd.

Waarom extra documentatie? Dit is toch al gespecificeerd in #1255

Reply:
Wijzig Document gemeente in En stap naar '0518'.

Reply:
In de En stap is 'gemeente document (82.10)' aangepast naar '0518'.

Reply:
> Wat maakt nu dat dit een ontkenning ouderschap is ? Dit zou toch ook een document kunnen zijn dat met Adoptie te maken heeft ?

Titel is inmiddels aangepast.